docs/parsing.md

```diff
@@ -74,4 +74,4 @@ for 4 nodes and no results from different node counts are included).
   * The elapsed time is found by taking the difference between the start time
 for the first epoch (Epoch 0) and the stop time for the last epoch.
   * All results are averaged together based on the number of results in the
-directory.
\ No newline at end of file
+directory.
```

Comment:
Weird. I didn't touch this line. Not intentionally, anyway. And I edited with vim...

Reply:
Wonder if it's a newline at the end difference?

Reply:
Looking directly at the commit, it does look like there's a newline at the end of this doc now which is expected, so I'd say this is good.